.github/workflows/ci.yml

```diff
@@ -64,5 +64,5 @@ jobs:
           cd KinFragLib
           # Run tests
           PYTEST_ARGS="--nbval-lax --current-env --nbval-cell-timeout=900"
-          pytest $PYTEST_ARGS notebooks/custom_kinfraglib/
+          pytest $PYTEST_ARGS notebooks/custom_kinfraglib/ --ignore /notebooks/custom_kinfraglib/2_1_custom_filters_pipeline.ipynb
```

Comment:
@sonjaleo, can you try to run this locally?
```
cd path/to/KinFragLib
PYTEST_ARGS="--nbval-lax --current-env --nbval-cell-timeout=900"
pytest $PYTEST_ARGS notebooks/custom_kinfraglib/ --ignore notebooks/custom_kinfraglib/2_1_custom_filters_pipeline.ipynb
```

Does it ignore the specified notebook on your computer?

Reply:
Yes, it ignores the notebook
`(kinfraglib) nona@DESKTOP-JJ6UBKL:~/masterthesis/KinFragLib$ PYTEST_ARGS="--nbval-lax --current-env --nbval-cell-timeout=900"
(kinfraglib) nona@DESKTOP-JJ6UBKL:~/masterthesis/KinFragLib$ pytest $PYTEST_ARGS notebooks/custom_kinfraglib/ --ignore notebooks/custom_kinfraglib/2_1_custom_filters_pipeline.ipynb
================================================= test session starts ==================================================
platform linux -- Python 3.8.12, pytest-5.4.3, py-1.11.0, pluggy-0.13.1
rootdir: /home/nona/masterthesis/KinFragLib
plugins: anyio-3.3.4, nbval-0.9.6, forked-1.3.0, xdist-1.34.0
collected 114 items

notebooks/custom_kinfraglib/1_1_custom_filters_unwanted_substructures.ipynb .......................              [ 20%]
notebooks/custom_kinfraglib/1_2_custom_filters_drug_likeness.ipynb ......................                        [ 39%]
notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb ......................                     [ 58%]
notebooks/custom_kinfraglib/1_4_custom_filters_pairwise_retrosynthesizability.ipynb ............................ [ 83%]
...................                                                                                              [100%]

=========================================== 114 passed in 711.10s (0:11:51) ============================================`

Reply:
Maybe the `/` is the problem then?
You pushed
`--ignore /notebooks/custom_kinfraglib/2_1_custom_filters_pipeline.ipynb`
instead of 
`--ignore notebooks/custom_kinfraglib/2_1_custom_filters_pipeline.ipynb`

Reply:
That was it! THANKS YOU!!! 👍 :)